extension-config.json

```diff
@@ -53,43 +53,24 @@
         "type": "number"
       }
     },
-    "gridBadgeRatio": {
+    "badgeWidth": {
       "type": "admin",
-      "default": {
-        "width": 50,
-        "height": 70.42
-      },
+      "default": "25",
       "destination": "frontend",
       "params": {
-        "label": "Image ratio for badges within a grid view",
-        "description": "Set height and width to configure a ratio for badges with product images in a grid view",
-        "type": "json"
-      }
-    },
-    "cardBadgeRatio": {
-      "type": "admin",
-      "default": {
-        "width": 40,
-        "height": 56.33
-      },
-      "destination": "frontend",
-      "params": {
-        "label": "Image ratio for badges within a carde view",
-        "description": "Set height and width to configure a ratio for badges with product images in a card view",
-        "type": "json"
+        "label": "Badge width percentage",
+        "description": "Badge width percentage to be used for all badge location rendering",
+        "type": "text"
       }
     },
-    "pdpBadgeRatio": {
+    "maxBadgeWidth": {
       "type": "admin",
-      "default": {
-        "width": 60,
-        "height": 84.5
-      },
+      "default": null,
       "destination": "frontend",
       "params": {
-        "label": "Image ratio for badges within a PDP",
-        "description": "Set height and width to configure a ratio for badges with product images on a PDP",
-        "type": "json"
+        "label": "Maximum Badge Width",
+        "description": "Maximum badge width can be set to px or %",
```

Comment:
The sdk reports an error if the % is in the description. When it is out the config file is created without issue.

![image](https://user-images.githubusercontent.com/11764719/74273583-5621f600-4cd6-11ea-88aa-81e5658304a6.png)

I am not sure it makes sense to allow the max-width to be set as a percent.